src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Tsc/Model/Metric/MetricLableValuesRequest.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.BasicAbility.Tsc.Model;
+
+public class MetricLableValuesRequest
+{
+    [JsonIgnore]
```

Comment:
Matches are single or multiple I don't think developers need to know

Reply:
It's suport many to query, in this case only can query single ,so only can set `Match` for single